packages/inspector/client/auto-imports.d.ts

```diff
@@ -39,7 +39,6 @@ declare global {
   const makeDestructurable: typeof import('@vueuse/core')['makeDestructurable']
   const markRaw: typeof import('vue')['markRaw']
   const nextTick: typeof import('vue')['nextTick']
-  const note: typeof import('@vueuse/core')['note']
```

Comment:
Note that I have no idea why this was modified. It was done automatically. I can remove it from the PR if you prefer.